test/verible.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env sh
+
+# Authors:
+#   Unai Martinez-Corral
+#
+# Copyright 2022 Unai Martinez-Corral <unai.martinezcorral@ehu.eus>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+cd $(dirname "$0")
+
+./_env.sh
+
+which verible-patch-tool
```

Comment:
These checks might be contributed to https://github.com/hdl/smoke-tests. Nonetheless, it's not a blocker to have this merged. Let me know if you want to address it.

Reply:
Sure, opened an PR: hdl/smoke-tests#7